_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
```

Comment:
No need to spend time on it. If you've done this before, that's good enough for me. I _haven't_ done it before and testing for empty in javascript is something that scares me :)